wait/host.go

```diff
@@ -587,24 +587,32 @@ func (a *HostAwaitility) WaitForTemplateUpdateRequests(namespace string, count i
 type NotificationWaitCriterion func(a *HostAwaitility, mur *toolchainv1alpha1.Notification) bool
 
 // WaitForNotification waits until there is a Notification available with the given name and the optional conditions
-func (a *HostAwaitility) WaitForNotification(name string, criteria ...NotificationWaitCriterion) (*toolchainv1alpha1.Notification, error) {
-	var notification *toolchainv1alpha1.Notification
+func (a *HostAwaitility) WaitForNotification(name string, criteria ...NotificationWaitCriterion) ([]*toolchainv1alpha1.Notification, error) {
+	var notification []*toolchainv1alpha1.Notification
```

Comment:
```suggestion
	var notifications []*toolchainv1alpha1.Notification
```
(change to `notifications` will need to be applied further down in this func)

Reply:
Good point! Updated it as part of this commit: fd66ead0da803a9206a4e0326c56ae73dec1d2a5